apps/pie-microsite/src/assets/styles/components/_list.scss

```
@@ -0,0 +1,53 @@
+@use '../fozzie-config' as f;
+
+.c-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.c-list.c-list--ordered {
```

I tend not to add the base to the modifier here, mainly as it makes the CSS more specific (just avoids hitting any specificity issues down the line). As it's a modifier, the name should imply that it shouldn't be used without the base class

```suggestion
.c-list--ordered {
```